main.js

```diff
@@ -0,0 +1,417 @@
+// data
+
+let items={
```

Comment:
why are we using `let` here?

Reply:
I have thought to implement add Item functionality on server side so we can update items but not implemented yet. And in the next push i will comment this because all items will be stored in local storage